attach/nv_attach_impl/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+include(../../cmake/cuda.cmake)
+
+add_library(
+    bpftime_nv_attach_impl STATIC
+    nv_attach_impl.cpp
+    nv_attach_private_data.cpp
+    # cuda_injector.cpp
```

Comment:
do we need to use this file?

Reply:
`cuda_injector.cpp` is a file written by @vickiegpt and intended for dynamic gpu attach, we may remove it now since we didn't have support for